deploy/olm-catalog/smart-gateway-operator/1.0.0/smartgateway.infra.watch_smartgateways_crd.yaml

```diff
@@ -0,0 +1,115 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: smartgateways.smartgateway.infra.watch
+spec:
+  group: smartgateway.infra.watch
+  names:
+    kind: SmartGateway
+    listKind: SmartGatewayList
+    plural: smartgateways
+    singular: smartgateway
+  scope: Namespaced
+  version: v2alpha1
+  subresources:
+    status: {}
+  versions:
+  - name: v2alpha1
```

Comment:
Should this still be v2alpha1 or stable?

Reply:
This is just the API version, and is unrelated to the versioning of the CSV :)

Reply:
...or the operator container image tag
...or the Smart Gateway container image tag
...or the application registry version

Reply:
...or the channel name

Reply:
Thanks!